src/main/java/com/example/mylib/MyLib/model/Book.java

```diff
@@ -36,4 +36,24 @@ public Book(String title, String author, Customer bookOwner){
     public String toString() {
         return String.format("Book[id=%s, title=%s, author=%s, bookOwner=%s, date=%s]", id, title, author, bookOwner.getFirstName(), date);
     }
+
```

Comment:
Replace with Lombok. If you are having trouble with it:
1. Install Lombok Plugin from repositories
2. Enable Annotations Processing from Project Settings -> Compiler options